wallet-sdk/src/testIntegration/kotlin/org/stellar/walletsdk/AnchorPlatformTest.kt

```diff
@@ -0,0 +1,123 @@
+@file:OptIn(ExperimentalCoroutinesApi::class)
+
+package org.stellar.walletsdk
+
+import io.ktor.client.*
+import io.ktor.client.request.*
+import kotlin.test.assertEquals
+import kotlin.test.fail
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import org.stellar.walletsdk.anchor.WithdrawalTransaction
+import org.stellar.walletsdk.asset.IssuedAssetId
+import org.stellar.walletsdk.auth.AuthToken
+import org.stellar.walletsdk.horizon.SigningKeyPair
+import org.stellar.walletsdk.horizon.sign
+import org.stellar.walletsdk.horizon.toTransferTransaction
+
+class AnchorPlatformTest {
+  private val USDC =
+    IssuedAssetId("USDC", "GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP")
+  private val JPYC =
+    IssuedAssetId("JPYC", "GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP")
```

Comment:
Both assets have the same issuer; not sure if it's intentional.

Reply:
Yes, it's test asset for AP. Issued by same (internally used) account